NSSNamespace/Initial_Bangazon_Site
# dropdowns for categories and subcategories now working

Consumers can now select categories and subcategories for the products they create. 

**Methods added:** GetSubCategories on the ProductsController 

**Files changed:**

Controllers/ProductsController.cs
ViewModels/CreateProductViewModel.cs
Views/Products/Create.cshtml
wwwroot/js/site.js
wwwroot/js/site.min.js

**TO TEST:** 
git fetch 
git checkout dy-DropSecondTry 

Choose a customer from the drop down. 
Click Sell Something. 
Fill out the fields and choose from the Category dropdown 
The text in the second dropdown will change 
Then check the options on the dropdown, they should correspond with the subcategories 
Add your product and check that the product in your database matches the correct ProductTypeID and ProductTypeSubCategoryId.
